### PR TITLE
Add locale Japanese

### LINKED ISSUE #105: Japanese Language Support

### PR DESCRIPTION
**Pull request by 312k**
I added Japanese!
Closes #105

### CODE REVIEW

File: lib/conf.dart [hunk 1]
```diff
@@ -287,6 +293,12 @@ List<Language> get languageList => [
           nameLocal: 'italiano',
           locale: const Locale('it'),
           localizations: AppLocalizationsIt()),
+      Language(
+          // 85 million
```
Comment (marchellodev) [inline lib/conf.dart:297]: From wikipedia: "Japanese is spoken by more than 120 million people".

Therefore, in the comment it should say 120 million, not 85 million. Also, this `Language` object should be moved up the array based on that, as we are sorting languages by the number of speakers.

File: lib/conf.dart [hunk 2]
```diff
@@ -287,6 +293,12 @@ List<Language> get languageList => [
...
+          name: 'japanese',
+          nameLocal: '日本語',
+          locale: const Locale('ja'),
+          localizations: AppLocalizationsIt()),
```
Comment (marchellodev) [inline lib/conf.dart:301]: ` AppLocalizationsIt()`=> ` AppLocalizationsJa()`

**Review by marchellodev — CHANGES_REQUESTED** [review]
Good job, thanks!

I have left a few comments that needs to be resolved before we can merge this PR.

**Comment by 312k** [comment]
Thank you!
I revised those mistakes.

**Comment by marchellodev** [comment]
Analytics is disabled when the app runs in the debug mode, so there was no need to do that :)

Thanks, I'll merge this now!

**Comment by marchellodev** [comment]
@312k 
Also, can you please transfer all the japanese strings to the crowdin?

https://crowdin.com/project/sharik

As far as I understand, there is no way to automate this. Also, we are managing all our translations there

**Comment by marchellodev** [comment]
@312k
Also, can you please translate the app store strings? :)

https://crowdin.com/project/sharik

**Pull request merged**